src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -1101,6 +1101,7 @@ convert_oneD_coordset_to_strip(const conduit::Node &coordset,
     else
     {
         copy_node_rename_first_child(coordset, dest, "values", "y", "x");
+        dest["values/x"].reset();
```

Comment:
`reset` isn't needed b/c set will take care of it if needed.

Reply:
Before I added the call to `reset()`, I was seeing `dest["values/x"]` hold the same values and dimension as `dest["values/y"]`, except elements 0 and 1 were 0.0 and 1.0.  Now much has changed, and after trying it without the call to `reset()` I see that the expected two values are there.

Removed the call to `reset()`.